src/Entity/OgRole.php

```diff
@@ -1,9 +1,5 @@
 <?php
 
-/**
```

Comment:
Yes this was changed recently, see https://www.drupal.org/node/2304909. According to the updated coding standards it is now actually no longer permitted to have an `@file` block for PSR-4 namespaced classes.